### ISSUE TITLE
mPaintFooterIndicator color doesn't change when changing mPaintFooterLine color

### ISSUE THREAD

**Issue by ChrisSmith** [JakeWharton/ViewPagerIndicator#12]
Title says it all.

Hot fix, change the footerIndicator color when changing the line color. (line 148 TitlePageIndicator.java)

``` java
public void setFooterColor(int footerColor) {
        mPaintFooterLine.setColor(footerColor);
        mPaintFooterIndicator.setColor(footerColor);
        invalidate();
    }
```

**Comment by JakeWharton** [comment]
BOOM

**Comment by JakeWharton** [comment]
Released with 1.0.1.
